ister_gui.py

```diff
@@ -81,7 +81,10 @@ def get_disk_info(disk):
     """Return dictionary with disk information"""
     info = {'partitions': []}
     cmd = ['/usr/bin/fdisk', '-l', disk]
-    output = subprocess.check_output(cmd).decode('utf-8')
+    try:
+        output = subprocess.check_output(cmd).decode('utf-8')
+    except:
+        pass
```

Comment:
Shouldn't we return early here instead of trying to parse the output?

Reply:
I was going to do that but decided that the code is already in theory handling empty output and I don't like returning early if I don't need to just for readability.

Reply:
What's wrong with explicitly returning early? We know we want to return the initialized empty `info` dict when this errors out, but a `pass` means we are expecting the rest of the function to be a no-op on the dict. I think returning `info` here can help prevent bugs if we modify the rest of the function at some point in the future.

Reply:
Alrighty, I'll update the patch.